samples/drivers/soc_flash_nrf/README.rst

```diff
@@ -7,10 +7,11 @@
 Overview
 ********
 
-This sample demonstrates using the :ref:`Flash API <flash_api>` on an SoC internal flash.
-The sample uses :ref:`Flash map API <flash_map_api>` to obtain device for flash, using
-DTS node label, and then directly uses :ref:`Flash API <flash_api>` to perform
-flash operations.
+This sample demonstrates using the :ref:`Flash API <flash_api>` on a SoC internal flash.
```

Comment:
nit: we actually tend to spell "an SoC" (an s-o-c) more often than not in the docs

```suggestion
This sample demonstrates using the :ref:`Flash API <flash_api>` on an SoC internal flash.
```